sites/all/themes/custom/ogp_theme/lib/components/modal.js

```diff
@@ -221,25 +221,25 @@ function setMapModalContent(id, type, countryId, countriesData) {
     });
     switch (type) {
       case 'actionPlan':
-        pushSmallModal(id, `irm_commitments?filter[country]=${countryId}`, countryData, 'commitments', 'themes covered', 'latest stories', '/stories', '');
+        pushSmallModal(id, `irm_commitments?filter[country]=${countryId}`, countryData, 'commitments', 'themes covered', 'latest stories', 'stories', '');
         break;
       case 'starred':
-        pushDefaultModal(id, `starredcommitments?filter[country]=${countryId}`, countryData, 'starred commitments', 'latest stories', '/stories', 'list', '');
+        pushDefaultModal(id, `starredcommitments?filter[country]=${countryId}`, countryData, 'starred commitments', 'latest stories', 'stories', 'list', '');
         break;
       case 'event':
-        pushDefaultModal(id, `events?filter[country]=${countryId}`, countryData, 'events', 'go to events', '/events', 'list', '');
+        pushDefaultModal(id, `events?filter[country]=${countryId}`, countryData, 'events', 'go to events', 'events-and-news', 'list', '');
         break;
       case 'commitment':
```

Comment:
I still have this not working. Map does not change when selector is changed. Also links in modal always go to /theme. Can this be addressed in this PR?